lib/auto-languageclient.js

```diff
@@ -34,6 +35,16 @@ export default class AutoLanguageClient {
   _linterDelegate: linter$V2IndieDelegate;
   _signatureHelpRegistry: ?atomIde$SignatureHelpRegistry;
 
+  /**
+   * Available if consumeBusySignal is called, ie the client package.json includes:
+   * ```json
+   * "consumedServices": {
+   *   "atom-ide-busy-signal": { "versions": { "0.1.0": "consumeBusySignal" } }
+   * }
+   * ```
+   */
```

Comment:
I think we should remove this from here - we don't explicitly call out any of the other services that need to be added.  We'll instead add busy signal to the example starter project.

Reply:
Ok I'll tone it down to just a note to help those that find it to be null